docs/image.md

```diff
@@ -106,6 +107,37 @@ build:
 
 See the kubernetes documentation on [setting environment variables](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/) and [resource limits and requests](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container) for more information.
 
+### <a id='notary-config'></a>Notary Configuration
+
+The optional `notary` field on the `image` resource can be used to configure [Notary](https://github.com/theupdateframework/notary) image singing.
+```yaml
+notary:
+  v1:
+    url: "https://example.com/notary"
+    secretRef:
+      name: "notary-secret"
+```
+- `v1.url`: The URL of the notary server.
+- `v1.secretRef.name`: A [secret](#notary-secret) containing the encrypted private key and private key password.
+
+#### Generate Signing Key
+To generate a singing key, use the following commands from the [Docker Content Trust](https://docs.docker.com/engine/security/trust/#signing-images-with-docker-content-trust) documentation:
+```shell script
+% export DOCKER_CONTENT_TRUST_SERVER=<notary-server-url>
```

Comment:
Is there a reason to recommend the docker cli here vs [notary](https://github.com/pivotal/kpack/blob/master/rfcs/0005-notary-integration.md#key-generation-and-storage)?

Reply:
Using the docker CLI is significantly easier. I also suspect thats what most users will use.